### PR TITLE
Update jenkins Dockerfile

### PR DESCRIPTION
**Pull request by kuanfandevops**
Updated the curl url for Openshift Client. Openshift build failed to unzip it.

### CODE REVIEW

File: cicd/jenkins-basic/docker/Dockerfile [hunk 1]
```diff
@@ -40,8 +40,9 @@ ENV BASH_ENV=/usr/local/bin/scl_enable \
 #OC command line
 RUN set -x && \
     mkdir /tmp/oc && \
-    curl -SL https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-client-linux-4.5.11.tar.gz -o /tmp/oc/openshift-origin-client-tools.tar.gz && \
-    tar -xzf /tmp/oc/openshift-origin-client-tools.tar.gz -C /tmp/oc && \
+    curl -SL https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.5.15/openshift-client-linux-4.5.15.tar.gz -o /tmp/oc/openshift-origin-client-tools.tar.gz && \
```
Comment (cvarjao) [inline cicd/jenkins-basic/docker/Dockerfile:43]: can you please update `OC_VERSION` (up to of this Dockerfile) and then use it as environment variable in this line?

**Comment by stephenhillier** [comment]
Can I propose updating the env var in another PR if that will help this change (and #56) get merged? I can do that if Kuan doesn't have time.

File: cicd/jenkins-basic/docker/Dockerfile [hunk 2]
```diff
@@ -40,8 +40,9 @@ ENV BASH_ENV=/usr/local/bin/scl_enable \
...
+    gunzip /tmp/oc/openshift-origin-client-tools.tar.gz && \
```
Comment (cvarjao) [inline cicd/jenkins-basic/docker/Dockerfile:44]: @kuanfandevops, can you please revert this back to `tar -xsf`. Also, with binaries from mirror.openshift.com, you can extract directly to `usr/local/bin` without going through the `/tmp` folder.

**Comment by BcGovNeal** [comment]
There is also the option to get oc binary from https://downloads-openshift-console.apps.silver.devops.gov.bc.ca/amd64/linux/oc.tar.  This way it should always match the cluster version.

File: cicd/jenkins-basic/docker/Dockerfile (hunk 1 above)
Comment (david-agahchen) [reply cicd/jenkins-basic/docker/Dockerfile:43]: Just created PR #57 based on recommendation from @cvarjao .

**Comment by kuanfandevops** [comment]
@cvarjao I missed the notification, just updated the settings in my account, future notifications should work fine

**Pull request merged**